src/program/snabbvmx/doc/README.troubleshooting.md

```diff
@@ -320,7 +320,34 @@ NOTE: Currently the test is not working correctly: the returned MAC should be
 `02:99:99:99:99:99`.
 
 
-## End-to-end tests - details
+## crafting and running end-to-end tests
```

Comment:
Other headers start with a capital letter, I would prefer to keep the same style in all headers.